invite/templates/invite/contact.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'invite/base.html' %}
+{% block title %}
+Account Inactive
+{% endblock %}
+{% block content %}
+Your account is inactive. Please contact {{ supervisor }} to reactivate the account.
```

Comment:
So I feel like we may want to rename this variable. The contact person may not necessarily be the supervisor of whoever is attempting a reset. Maybe just rename it `reset_contact`? Also, I think it would be helpful to actually have an email link here, not just a name, to make the process of getting in contact with that individual easier.